akka-docs/rst/general/addressing.rst

```diff
@@ -143,6 +139,18 @@ systems or JVMs. This means that the logical path (supervision hierarchy) and
 the physical path (actor deployment) of an actor may diverge if one of its
 ancestors is remotely supervised.
 
+
+Actor path alias or symbolic link?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+As in some real file-systems you might think of a "path alias" or "symbolic link”for an actor,
```

Comment:
Seems we're using the "proper" quotes through the document, please use `“...”` consistently then, once that's fixed (the `"` before symbolic should match the ending one, and a space after `”`) we'll merge.

Sorry for the trouble :)